tests/unittest_brain_turtle.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2020 rioj7
+
+from textwrap import dedent
+import unittest
+
+import astroid
+
+class TurtleTest(unittest.TestCase):
+
+    def test_pencolor(self):
+        node = dedent("""
+            import turtle
+            turtle.pencolor
+            """)
+        inferred = next(astroid.extract_node(node).infer())
+        self.assertIsInstance(inferred, astroid.FunctionDef)
+        self.assertEqual(inferred.name, 'pencolor')
```

Comment:
Why just focusing on `pencolor` function?
It should be possible to loop over all the function you added in the brain and make for each of them the same tests you are doing here.